doc/nrf/device_guides/working_with_nrf/nrf53/nrf5340.rst

```diff
@@ -579,6 +579,49 @@ Bluetooth buffers configuration introduced by the :kconfig:option:`CONFIG_NCS_SA
    :start-after: fota_upgrades_zigbee_start
    :end-before: fota_upgrades_zigbee_end
 
+.. _ug_nrf5340_image_slot_numbering:
+
+Image IDs and slot numbering
+****************************
+
+Each image in the DFU process on the nRF5340 DK is identified by a unique ID, corresponding to a specific memory slot for each core.
+The slots are numbered from 0, and each slot corresponds to a specific location in memory.
+This configuration typically follows:
+
++-------+-------------------------------------------------------------+-----------+-----------+
+| Slot  | Description                                                 | DTS label | n <image> |
++=======+=============================================================+===========+===========+
+| slot0 | First image's primary slot (internal flash)                 | image-0   | 1         |
++-------+-------------------------------------------------------------+-----------+-----------+
+| slot1 | First image's secondary slot (internal or external flash)   | image-1   | 0, 2      |
++-------+-------------------------------------------------------------+-----------+-----------+
+| slot2 | Second image's primary slot (internal RAM)                  | image-2   | 3         |
++-------+-------------------------------------------------------------+-----------+-----------+
+| slot3 | Second image's secondary slot (internal or external flash)  | image-3   | 4         |
++-------+-------------------------------------------------------------+-----------+-----------+
+
+.. note::
+   Slot 1 is the default upload target for firmware updates using the mcumgr CLI or similar software.
+   This behavior applies when no image number is provided and the application is build with the ``CONFIG_MCUBOOT_SERIAL_DIRECT_IMAGE_UPLOAD`` Kconfig option enabled.
```

Comment:
CONFIG_MCUBOOT_SERIAL_DIRECT_IMAGE_UPLOAD is for the bootloader. While this says "the application is built with".
Do we mean that MCUboot is built with this, or do we mean to say CONFIG_MCUMGR_GRP_IMG_DIRECT_UPLOAD instead?